### ISSUE TITLE
[Marketplace] Clicking Account Management as marketplace owner for shops doesn't work

### ISSUE THREAD

**Issue by cindyfirestone** [reactioncommerce/reaction#2726]
### Expected behavior
As a marketplace owner, when I go to the dropdown of shops on my marketplace and select a shop, I should be able to click on accounts to see users of that shop and their permissions. 

### Actual Behavior
Nothing happens 

### Steps to Reproduce the Behavior
1. Login as admin
2. Choose a shop from the dropdown in the upper left side of the page
3. Click on Accounts in dashboard 
4. See spinning circle 

### Versions
```
Node: 7.2.0
NPM: 5.3.0
Meteor Node: 4.8.4
Meteor NPM: 4.6.1
Reaction CLI: 0.11.0
Reaction: 1.5.0
Reaction branch: marketplace
Docker: 17.06.1-ce
```

**Comment by mikemurray** [comment]
Some console errors for this:

```
TypeError: Cannot read property '_id' of undefined
    at AdminInviteForm.renderDropDownButton (adminInviteForm.js:99)
    at AdminInviteForm.renderForm (adminInviteForm.js:166)
    at AdminInviteForm.render (adminInviteForm.js:194)
    at modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:108479
    at measureLifeCyclePerf (modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:107759)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:108478)
    at ReactCompositeComponentWrapper._renderValidatedComponent (modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:108505)
    at ReactCompositeComponentWrapper.performInitialMount (modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:108045)
    at ReactCompositeComponentWrapper.mountComponent (modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:107941)
    at Object.mountComponent (modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:100702)
modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:97214 Uncaught TypeError: Cannot read property '__reactInternalInstance$fnvx84k4k0a' of null
    at Object.getClosestInstanceFromNode (modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:97214)
    at findParent (modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:110726)
    at handleTopLevelImpl (modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:110755)
    at ReactDefaultBatchingStrategyTransaction.perform (modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:101667)
    at Object.batchedUpdates (modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:110673)
    at Object.batchedUpdates (modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:100335)
    at dispatchEvent (modules.js?hash=e1dc43198559d8c83210ed3bd5544d5726dec5ba:110835)
```

**Comment by impactmass** [comment]
working on this now

**Comment by impactmass** [comment]
Fixed by #2734
